cmscontrib/AddSubmission.py

```diff
@@ -33,17 +33,26 @@
 import logging
 import sys
 
-from cms import utf8_decoder
+from cms import utf8_decoder, ServiceCoord
 from cms.db import File, Participation, SessionGen, Submission, Task, User, \
     ask_for_contest
 from cms.db.filecacher import FileCacher
 from cms.grading.languagemanager import filename_to_language
+from cms.io import RemoteServiceClient
 from cmscommon.datetime import make_datetime
 
 
 logger = logging.getLogger(__name__)
 
 
+def maybe_send_notification(submission_id):
+    """Non-blocking attempt to notify a running ES of the submission"""
+    rs = RemoteServiceClient(ServiceCoord("EvaluationService", 0))
+    rs.connect()
+    rs.new_submission(submission_id=submission_id)
+    rs.finalize("Closing script")
```

Comment:
You may want to call `disconnect()` instead, as `finalize` is more lower-level...